interface/github.py

```diff
@@ -45,6 +45,49 @@ def org_has_member(self, username):
         except GithubException as e:
             raise GithubAPIException(e.data)
 
+# ---------------------------------------------------------------
+# --------------- methods related to team members ---------------
+# ---------------------------------------------------------------
+
+    def list_team_members(self, team_id):
+        """Return a list of users in the team of id team_id."""
+        try:
+            team = self.github.get_team(team_id)
+            # Question: Should we return github.PaginatedList.PaginatedList or just regular python's list
```

Comment:
On this, check with @jacqueschen1 - I believe he converts `PaginatedList` to a standard list, but I can't quite remember.

Reply:
yea you can iterate through the `PaginatedList` and put everything in a Python list.